scripts/heartbeat-fix.sh

```diff
@@ -4,6 +4,6 @@
 
 SSO_RESP=$(curl "${SSO_URL}" -H 'Content-Type: application/x-www-form-urlencoded' --data-urlencode 'grant_type=client_credentials' --data-urlencode "client_id=${SA_CLIENT_ID}" --data-urlencode "client_secret=${SA_CLIENT_SECRET}")
 TOKEN=$(echo $SSO_RESP | awk -F ':"' '{print $2}' | cut -d '"' -f 1);
-INVENTORY_URL="kessel-inventory-api:8000/api/inventory/v1beta2/resources";
+INVENTORY_URL="https://kessel-inventory-api:8000/api/inventory/v1beta2/resources";
```

Comment:
since ENV_NAME should be stage and prod i think we can set this to 
`INVENTORY_URL="https://kessel-inventory-api.kessel-${ENV_NAME}.svc:8000/api/inventory/v1beta2/resources";`
and that will fix the DNS issues hopefully, i had the same issue in KIC with resolution

Hopefully `--insecure` works, if it doesnt i can go back and poke at this, the cronjob isnt too critical at this time i dont think

Reply:
The cronjob has run about 3 times and hasn't had any issues sending requests. Looks like events are being consumed properly.